shared/src/main/java/uk/gov/di/authentication/shared/state/StateMachine.java

```diff
@@ -7,12 +7,7 @@
 import uk.gov.di.authentication.shared.entity.SessionState;
 import uk.gov.di.authentication.shared.services.ConfigurationService;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
should this be a * i think somewhere in the gds way it says no star imports?

Reply:
Well spotted ... commit pushed!